src/components/calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './calculator.css';
+
+class Calculator extends React.Component {
```

Comment:
Please not that it's required that the Calculator.js component should be a functional component 🤝🏽